content/blog/multi-lang-components/index.md

```diff
@@ -14,29 +14,31 @@ tags:
     - typescript
 ---
 
-Today we're excited to announce support for one of our most highly-requested features: multi-language components! Our
-first multi-language component is the popular [EKS package], which was previously only available for TypeScript. With the
-[v0.22.0 release], teams can [manage EKS clusters] in their language of choice (Python, .NET, Go, and TypeScript).
+Pulumi's infrastructure as code tooling combines the programming languages and tools you already know with the full power of cloud
+infrastructure. But, until now, some of that Pulumi components for that cloud infrastructure, like our popular [EKS package], were
+only available in a subset of the languages supported by Pulumi.
 
-<!--more-->
+Now, with multi-language components, anyone can author a Pulumi component once, in the language they prefer, and make it available
+across all Pulumi languages. To celebrate, we've taken that [EKS package]–previously only available for TypeScript–
+and brought it to all four Pulumi languages: TypeScript, Python, .NET, and Go. Now, regardless of the language you choose, you can
```

Comment:
and made it available to all four Pulumi languages: TypeScript, Python, .NET, and Go. Regardless of the language you choose, you can